assets/templates/kubernetes.container/gotext_multiline.tpl

```diff
@@ -0,0 +1,559 @@
+{{- $period := generate "metricset.period" }}
+{{- $agentId := generate "agent.id" }}
+{{- $agentVersion := generate "agent.version" }}
+{{- $agentName := generate "agent.name" }}
+{{- $agentEphemeralid := generate "agent.ephemeral_id" }}
+{{- $rxbytes := generate "container.network.ingress.bytes" | int }}
+{{- $txbytes := generate "container.network.egress.bytes" | int }}
+{{- $uId := uuidv4 }}
+{{- $pod_uId := uuidv4 }}
+{{- $container_uId := uuidv4 }}
+{{- $suffix := split "-" $uId }}
+{{- $timestamp := generate "timestamp" }}
+{{- $fulltimestamp := $timestamp.Format "2006-01-02T15:04:05.999999Z07:00" }}
+{{- $resttime := split ":" $fulltimestamp }}
+{{- $picktimedate := generate "timedate" }}
+{{- $timehour := generate "timehour" }}
+{{- $offset := generate "Offset" | int }}
+{{- $faults := generate "faults" | int }}
+{{- $pct := generate "Percentage" | float64 }}
```

Comment:
if you `divf $pct` you should not need to cast it to `float64`: https://github.com/Masterminds/sprig/blob/master/numeric.go#L178

Reply:
Fixed in 31b2839e62064fc4d20c24fd78079e161efa8d06